go/vt/workflow/resharding/parallel_runner.go

```diff
@@ -133,7 +133,9 @@ func (p *ParallelRunner) Run() error {
 		}
 		select {
 		case <-p.ctx.Done():
-			log.Info("Workflow is cancelled, will not run task: %v", task)
+			log.Infof("Workflow is cancelled, will not run task: %v", task)
```

Comment:
This log message is somewhat misleading since it will not run this task _or_ all subsequent ones, right?

Reply:
Yeah that is correct. How would you phrase it? 

Maybe just:

```
log.Infof("Workflow is cancelled, current task and subsequent tasks will be aborted")  
```

Reply:
Sure